### ISSUE TITLE
Errors while closing the simulation viewport

### ISSUE THREAD

**Issue by kalwalt** [augmentmy-world/arStudio#5]
When the simulation is stopped an error message is showed in the console:

```js
 + FINISH 
arcontroller.compone…js?version=0.64:306 Stop AR
arcontroller.compone…js?version=0.64:311 Uncaught TypeError: Cannot read property 'getTracks' of null
    at ArControllerComponent.stopAR (arcontroller.compone…js?version=0.64:311)
    at Object.trigger (litegl.js?version=0.64:10617)
    at Scene.finish (litescene.js?version=0.64:14467)
    at Object.changeState (play.js?version=0.64:300)
    at Object.onPlay (play.js?version=0.64:128)
```

**Comment by ThorstenBux** [comment]
Yeah, I know. the UserMediaAPI has changed in the mean time. We need to fix that.
Also the arCamera node is not removed after stoping the scene which leads to these nodes piling up

**Comment by kalwalt** [comment]
yes i saw them, and it make difficult to manage the nodes too.

**Comment by ThorstenBux** [comment]
I had that issue once so I think that is solvable

**Comment by kalwalt** [comment]
This issue was caused because the video was deleted by calling dispose() https://github.com/augmentmy-world/arStudio/blob/08583457e9969023168b86d0eefd24f37a151659/editor/js/components/arcontroller.component.js#L308 inside stopAr() as a consequence the video src was null and the the `this._video.srcObject.getTracks()[0].stop();` found anything to remove. That commit solve also the other issue related to arCamera.

**Comment by kalwalt** [comment]
Solved with #6
